beta/google-cloud-run/README.md

```diff
@@ -0,0 +1,236 @@
+# Deploy 1Password SCIM Bridge on Google Cloud Run
+
+_Learn how to deploy 1Password SCIM Bridge on [Cloud Run](https://cloud.google.com/run/docs/overview/what-is-cloud-run) using the Cloud Shell in Google Cloud._
+
+This guide can be used to deploy 1Password SCIM Bridge as an ingress container to a single replica [Cloud Run service](https://cloud.google.com/run/docs/overview/what-is-cloud-run#services) with the required Redis cache deployed as a sidecar container. Credentials are stored in Secret Manager and mounted as volumes attached to the SCIM Bridge container.
+
+The included [Cloud Run service YAML](https://cloud.google.com/run/docs/reference/yaml/v1#service) manifests should be suitable for use in a production environment without modification, but are intentionally minimal for simplicity, to allow any identity provider to connect to its public endpoint, and to facilitate its use as base for a customized deployment.
+
+*Table of contents:*
+
+- [Before you begin](#before-you-begin)
+
+<!-- !TODO: Complete ToC  -->
+
+## Before you begin
+
+<!-- !TODO: Document prereqs  -->
+
+## Step 1: Set up Google Cloud
+
+1. Create a [project](https://cloud.google.com/docs/overview#projects) to organize the Google Cloud resources for your 1Password SCIM Bridge deployment, and set it as the default project for your Cloud Shell environment:
+
+    ```sh
+    gcloud projects create op-scim-bridge --set-as-default
+    ```
+
+2. Enable the Secret Manager and Cloud Run APIs for your project:
+
+    ```sh
+    gcloud services enable secretmanager.googleapis.com run.googleapis.com
+    ```
+
+## Step 2: Create a secret for your `scimsession` credentials
+
+The Cloud Run service for SCIM Bridge will be configured to mount volume using a secret from Secret Manager. Follow these steps to upload your `scimsession` credentials file to the Cloud Shell, create a secret, and store the file contents as its first secret version:
+
+1. Click **⋮** _(More)_ > **Upload** in the Cloud Shell terminal menu bar.
+2. Click **Choose Files**. Select the `scimsession` file that you saved to your computer.
+3. Use the suggested destination directory. Click **Upload**.
+> [!NOTE]
+> If the file is saved to a different directory or using a different file name, make a note of the full path to
+> the file.
+4. Create a secret with the contents of this file as its first secret version using the following command:
+
+    ```sh
+    gcloud secrets create scimsession --data-file=$HOME/scimsession
+    ```
+
+> [!TIP]
+> The command above is expected work as is if the file is named `scimsession` and if it was saved to the home
+> directory when uploading the file. If not, replace `$HOME/scimsession` with the actual path to the file. For
+> example:
+>
+> ```sh
+> gcloud secrets create scimsession --data-file=/example/path/to/scimsession.file
+> ```
+
+5. Enable Cloud Run to access the secret using the Compute Engine default service account for the project:
```

Comment:
The formatting of this command or just the command itself did not work on a customers environment. Perhaps there is another piece that needs enabling before we can run this command.

Reply:
There were missing quotes surrounding the format mask used to find the Compute Engine default service account for the project.